app/javascript/lib/validation/expression-matrices-validation.js

```diff
@@ -185,10 +185,9 @@ function getDenseMatrixDelimiter(rawHeader, rawNextTwoLines) {
  *
  * The "cap" for an expression matrix file is the first row also called the "header"
  *
- * A dense matrix header must start with the value 'GENE' or if the file is R-formatted it can:
- *  - Not have GENE in the header and:
- *    - Have one less entry in the header than each successive row OR
- *    - Have "" as the last value in header.
+ * A dense matrix header must start with the value 'GENE' or if the file is R-formatted it can
```

Comment:
We conventionally double-quote this kind of user-facing text:

```suggestion
 * A dense matrix header must start with the value "GENE" or if the file is R-formatted it can
```